content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -24,10 +24,35 @@ The following steps require an egress configuration, for example:
 You need to configure the API Server to use the Konnectivity service
 and direct the network traffic to the cluster nodes:
 
+1. Make sure that
+the `ServiceAccountTokenVolumeProjection` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+is enabled. You can enable
+[service account token volume protection](/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
+by providing the following flags to the kube-apiserver:
+   ```
+   --service-account-issuer=api
+   --service-account-signing-key-file=/etc/kubernetes/pki/sa.key
+   --api-audiences=system:konnectivity-server
```

Comment:
@Jefftree I don't remember setting this kube-apiserver flag when set up konnectivity in GKE. Do you know why it's not required in GKE?

Reply:
If you're referring to the set of flags, they should be set in the GCE script [here](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-kubeapiserver.sh#L138-L140). For this specific flag, GCE seems to have `service-account-api-audiences` set to the issuer URL instead of the full list of audiences (I'm assuming that encompasses the system:konnectivity-server audience?). Definitely +1 on adding this section about ServiceAccountTokenVolumeProjection though.

cc @mikedanese for more context around these flags.